### PR TITLE
Sgar 986 manifest details ui changes

### PR DESCRIPTION
**Pull request by chanc789**
**What changes does this PR bring?**
Change labels and captions on manifest page to make instructions more clear to end user.

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [x] Is the build confirmed to be passing on CI?
- [x] Have you added enough relevant unit tests for your change?
- [x] Have you added enough relevant E2E tests for your change?
- [x] Have you updated any relevant documentation as part of this change?
- [x] Has this change been tested against the Acceptance Criteria?
- [x] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

File: src/app/garfile/manifest/addnewperson/index.njk [hunk 1]
```diff
@@ -26,24 +26,18 @@
...
-      </p>
+      <h1 class="govuk-heading-xl">{{__('heading_manifest_add_person_details')}}</h1>{# Add new person details #}
+      <p class="govuk-body-l">{{__('caption_manifest_person_details')}}</p>
 
       <form action="/garfile/manifest/addnewperson" autocomplete="off" class="form-group" method="post" role="form">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
         {% include "person-details.njk" %}
         <br>
-        <input id="add-person-manifest" class="govuk-button" type="submit" value="Add to GAR" onclick="sendAnalytics(event, 'Save and Exit', 'click')">
+        <input id="add-person-manifest" class="govuk-button" type="submit" value="Save and add to GAR" onclick="sendAnalytics(event, 'Save and Exit', 'click')">
```
Comment (nguyendhod) [inline src/app/garfile/manifest/addnewperson/index.njk]: How come no new "form_save_and_add_to_gar" in the en.json?

File: src/locales/en.json [hunk 2]
```diff
@@ -78,7 +78,7 @@
     "create_gar_caption_2": ".",
     "button_start_now": "Start now",
     "heading_manage_gars": "Manage GARs",
-    "manage_gars_caption": "You can view, edit, cancel or delete GARs dependent on the status of the GAR.",
+    "manage_gars_caption": "You can view, edit, cancel or delete GARs depending on the status of the GAR. Only an unsubmitted GAR can be cancelled. You cannot edit a GAR once it has been submitted.",
```
Comment (nguyendhod) [inline src/locales/en.json]: Lol, I previously took some stuff out, but if this describes it better, then that's ok with me.

File: src/app/garfile/manifest/index.njk [hunk 3]
```diff
@@ -135,38 +134,40 @@
...
-															</tr>
-                              <tr class="govuk-table__row">
-                                <td class="govuk-table__cell" colspan="7">
-                                  <button class="link btn-link" name="deletePersonId" value="{{person.garPeopleId}}">Delete</button>
-                                </td>
-                              </tr>
-														</tbody>
-													</table>
-												</div>
+											<summary role="button" aria-controls="details-content-2" aria-expanded="false" title="Further information for {{person.firstName}} {{person.lastName}}">
```
Comment (nguyendhod) [inline src/app/garfile/manifest/index.njk]: Externalised?

"further_information_for": "Further information for {{ givenName}} {{ surname }}"

and then I think its:

{{__('further_information_for', { givenName: person.firstName, surname: person.lastName }) }}

File: src/app/garfile/manifest/index.njk [hunk 4]
```diff
@@ -135,38 +134,40 @@
...
+                        <span class="summary">{{__('label_manifest_further_personal_details')}}</span>
+                      </summary>
+											<div class="panel panel-border-narrow" id="details-content-2" aria-hidden="true">
+												<table class="govuk-table" width="100%">
+													<tbody class="govuk-table__body">
+														<tr class="govuk-table__row">
+															<td class="govuk-table__cell">Gender</td>
```
Comment (nguyendhod) [inline src/app/garfile/manifest/index.njk]: These already have externalised strings from the person-details.njk I think?

File: src/app/garfile/manifest/index.njk (hunk 3 above)
Comment (chanc789) [reply src/app/garfile/manifest/index.njk]: Done

File: src/app/garfile/manifest/index.njk (hunk 4 above)
Comment (chanc789) [reply src/app/garfile/manifest/index.njk]: Done

File: src/locales/en.json (hunk 2 above)
Comment (nguyendhod) [reply src/locales/en.json]: Yeah, SGAR-715 says that submitted GARs can be cancelled, so I think that sentence should be removed.

**Pull request merged**